DEV_GUIDE.md

```diff
@@ -226,10 +226,17 @@ podman push quay.io/<your-username>/<repository-name>:<tag>
 Alternatively, to test locally made changes, push the built operator and proxy images into your Minikube.
 
 ```
-minikube image load kroxylicious-operator/target/kroxylicious-operator.img.tar.gz
-minikube image load kroxylicious-app/target/kroxylicious-proxy.img.tar.gz
+minikube image load kroxylicious-operator/target/kroxylicious-operator.img.tar.gz --alsologtostderr=true 2>&1 | tail -n1
+minikube image load kroxylicious-app/target/kroxylicious-proxy.img.tar.gz --alsologtostderr=true 2>&1 | tail -n1
 ```
 
+> :warning: Some minikube container runtimes may not be able to load a gzipped tar, if the above commands report a failure 
```

Comment:
Let's note the issue number so that it is easy to remove the advice if the issue gets fixed.

```suggestion
> :warning: Some minikube container runtimes may not be able to load a gzipped tar (https://github.com/kubernetes/minikube/issues/21678), if the above commands report a failure 
```